src/sentry/relay/projectconfig_cache/redis.py

```diff
@@ -63,6 +68,11 @@ def get(self, public_key):
                 rv = zstandard.decompress(rv_b).decode()
             except (TypeError, zstandard.ZstdError):
                 # assume raw json
-                pass
+                rv = rv_b
```

Comment:
Drive by fix ... That would have just blown up before because `rv` is not defined.